extensions/cl_intel_unified_shared_memory.asciidoc

```diff
@@ -721,7 +722,11 @@ Otherwise, it will return one of the following error values:
       | Returns allocation flags for the Unified Shared Memory allocation.
 
         Returns `0` if no allocation flags were specified for the Unified Shared Memory allocation and for `CL_MEM_TYPE_UNKNOWN_INTEL` allocations.
+| `CL_MEM_ALLOC_BUFFER_LOCATION_INTEL`
+  | cl_intel_mem_alloc_buffer_location
```

Comment:
Does the `cl_intel_mem_alloc_buffer_location` type need defined, or is it defined somewhere already? Think `intel` part should also be moved so it's a suffix of the type.

Reply:
`cl_intel_mem_alloc_buffer_location` is defined in `cl_intel_mem_alloc_buffer_location.asciidoc`, I kept intel to be at the beginning because all other extensions eg. `cl_intel_mem_channel_property.asciidoc` uses `intel` before type (specifically it uses `cl_intel_mem_channel_property`).

Reply:
The naming convention for the extension name itself is all good, but the convention for types is different and is to have the suffix at the end. E.g `cl_mem_properties_intel`, `cl_mem_alloc_flags_intel`.  So i'd expect to see this similarly defined somewhere like `typedef foo cl_mem_alloc_buffer_location _intel`

Reply:
Thank you, I have changed it to align with the types in `cl_intel_mem_alloc_buffer_location.asciidoc`